clients/instance/ibm-pi-dhcp.go

```diff
@@ -12,19 +12,19 @@ import (
 	"github.com/IBM-Cloud/power-go-client/power/models"
 )
 
-// NewIBMPIDhcpClient ...
+// NewIBMPIDhcpClient
 type IBMPIDhcpClient struct {
 	IBMPIClient
 }
 
-// NewIBMPIDhcpClient ...
+// NewIBMPIDhcpClient
 func NewIBMPIDhcpClient(ctx context.Context, sess *ibmpisession.IBMPISession, cloudInstanceID string) *IBMPIDhcpClient {
 	return &IBMPIDhcpClient{
 		*NewIBMPIClient(ctx, sess, cloudInstanceID),
 	}
 }
 
-// Create
+// Create a DHCP
```

Comment:
The swagger file says (OpenShift Internal Use Only) for all DHCP endpoints so should this be here at all?
If so this should be "Create a DHCP server"